sections/semantics-sections.include

```diff
@@ -293,10 +293,20 @@
     appropriate only if the element's contents would be listed explicitly in the document's
     <a>outline</a>.
   </p>
+  
+  <p>The semantics of the <{section}> element may be conveyed to users of Assistive Technology when 
+  the element has an explicit label. This information can provide a hint to users as to the type of content. 
+  For example the <a>role</a> of the element, which in this case is "region", 
+  can be announced by screen reader software when a user navigates to an <{section}> element. User Agents 
+  may also provide methods to navigate to <{section}> elements.</p>
 
   <div class="example">
     In the following example, we see an article (part of a larger Web page) about apples,
-    containing two short sections.
+    containing two short sections. 
+	
+	<p class="note">As the <{section}> has an <code>aria-label</code> attribute providing a brief description of 
+	the contents. Assistive technology may convey the <a value for="role"><code>region</code></a> role 
```

Comment:
Grammar nit. Either 

> …contents, assistive…

and use "can" not "may",

Or remove "As" from the beginning of the sentence.